biz.aQute.repository/src/aQute/bnd/repository/osgi/OSGiIndex.java

```diff
@@ -64,6 +64,15 @@
 		return bridge;
 	}
 
+	/**
+	 * Return the bridge repository in {@link Promise} form.
+	 * 
+	 * @return promise representing the repository
+	 */
+	public Promise<BridgeRepository> getBridgeRepository() {
```

Comment:
Does this need to be `public`? I would prefer default access (package private) to avoid creating new API.